filers/default/src/main/java/org/commonjava/indy/filer/def/DefaultGalleyStorageProvider.java

```diff
@@ -82,6 +83,10 @@
     @Inject
     private CacheHandle<String, String> nfsOwnerCache;
 
+    @FastLocalFileRemoveCache
+    @Inject
+    private CacheHandle<String, ConcreteResource> fastLocalFileRemoveCache;
```

Comment:
Is ConcreteResource serializable?

**UPDATE:** It looks like whether ConcreteResource is serializable depends on the Location implementation it wraps. In our case, most of these are probably RepositoryLocation at worst...and it's serializability depends on whether RemoteRepository is serializable, which it is.